torch/_dynamo/utils.py

```diff
@@ -56,6 +56,7 @@
 from typing_extensions import Literal, TypeGuard
 
 import torch
+import torch._C._instruction_counter as i_counter
```

Comment:
We only use this twice in this file - I wouldn't bother with such a short name. Maybe just do the import without an alias and call `_instruction_counter.blah` directly?